regression/goto-diff/java-scope-change2/test.desc

```diff
@@ -0,0 +1,12 @@
+KNOWNBUG
+new.jar
+old.jar --json-ui
+// Enable multi-line checking
+activate-multi-line-match
+EXIT=0
+SIGNAL=0
+  "deletedFunctions": \[ \],
+  "modifiedFunctions": \[\n    {\n      "name": "java::Test\.foo:\(I\)I",\n      "sourceLocation": {\n        "file": "Test\.java",\n        "function": "java::Test\.foo:\(I\)I",\n        "line": "5"\n      }\n    }\n  \],
+  "newFunctions": \[ \],
+--
+^warning: ignoring
```

Comment:
Would it be possible to include a comment that says what's currently going wrong here, i.e., which of the expected patterns is not showing up?

Reply:
Good point, I have added a couple of comments in the file.

Reply:
Would you mind moving the comments to the end of the file, after another `--`? That would keep the file syntactically valid (with regard to the syntax expected and documented by the Perl script).

Reply:
No problem. should be done in the last upload.